cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -67,8 +67,18 @@ import qualified Ouroboros.Storage.ChainDB as ChainDB
 
 import           Cardano.Config.Protocol (TraceConstraints)
 import           Cardano.Config.Types
+import           Cardano.Tracing.ToObjectOrphans.BlockFetchServerEventTrace ()
+import           Cardano.Tracing.ToObjectOrphans.ChainDBTrace ()
+import           Cardano.Tracing.ToObjectOrphans.ChainSyncEventTrace ()
+import           Cardano.Tracing.ToObjectOrphans.ConsensusToObjectOrphans
+import           Cardano.Tracing.ToObjectOrphans.DnsTrace ()
+import           Cardano.Tracing.ToObjectOrphans.ErrorPolicyTrace ()
+import           Cardano.Tracing.ToObjectOrphans.ForgeEventTrace ()
+import           Cardano.Tracing.ToObjectOrphans.LabelPeerTrace ()
+import           Cardano.Tracing.ToObjectOrphans.MuxTrace ()
+import           Cardano.Tracing.ToObjectOrphans.SubscriptionTrace ()
+import           Cardano.Tracing.ToObjectOrphans.TxSubmissionTrace ()
 import           Cardano.Tracing.MicroBenchmarking
-import           Cardano.Tracing.ToObjectOrphans
```

Comment:
I would just keep ToObjectOrphans.hs that imports/exports all the others

Reply:
Frankly, I second this.  The way the change currently looks like is it's adding work in future with no obvious net gain.